reconfiguration/cmf/concord.cmf

```diff
@@ -414,28 +410,6 @@ Msg PruneLegacyConfiguration 78 {
   uint32 tick_period_seconds
```

Comment:
Why do we need to keep this ?

Reply:
PruneRequest has number of fields out of which I've kept these 2 even though they are not required anymore.

 # TicksGenerator period in seconds.
  uint32 tick_period_seconds

  # The number of blocks in a pruning batch.
  uint64 batch_blocks_num 
  
  
  I am planning to do cleanup these in next PR.